_lectures/lec06.md

```diff
@@ -168,3 +168,19 @@ int main()
     return 0;  // Make sure this is outside of the while loop, otherwise, the program will exit too soon
 }
 ```
+# Practice Questions
+1. Write a Makefile so that when we type `make hello`, the compiler will compile the file `main.cpp` into an executable named `program01`
+2. Write the following if/else-if/else chain as a switch statement
+```
+char grade;
+cin >> grade;
+if(grade == ‘A’)
+	cout << “Grade > 90%” << endl;
+else if(grade == ‘B’)
+	cout << “Grade > 80%” << endl;
+else if(grade == ‘C’)
+	cout << “Grade > 70%” << endl;
+else
+	cout << “Shamefur dispray” << endl; 
```

Comment:
Can this be changed to something more encouraging that helps with the "growth mindset"?

Reply:
Just changed it into vim commands